### PR TITLE
Fixed losses docs

### PR DESCRIPTION
**Pull request by abhaikollara**
- Renamed objectives.md -> losses.md
- Replaced the terms "objective" with "loss"
- Changed `(nb_sample, nb_classes)` to  `(num_samples, num_classes)`
- Removed losses.md from auto-generation since no content is auto-generated

### CODE REVIEW

File: docs/templates/losses.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
-## Usage of objectives
+## Usage of losses
 
-An objective function (or loss function, or optimization score function) is one of the two parameters required to compile a model:
+An loss function (or objective function, or optimization score function) is one of the two parameters required to compile a model:
```
Comment (rilut) [inline docs/templates/losses.md]: `A loss` 😄

Comment (abhaikollara) [inline docs/templates/losses.md]: Yeah, my bad. I told myself it's not a good idea to use find and replace 😁

Comment (abhaikollara) [inline docs/templates/losses.md]: Yeah, my bad. I told myself it's not a good idea to use find and replace 😁

**Comment by fchollet** [comment]
Oh, I had actually updated this one yesterday, but somehow I had forgotten to push the files. Anyway, thank you!

**Comment by abhaikollara** [comment]
Always happy to help !

**Comment by abhaikollara** [comment]
@fchollet It still isn't showing the available loss functions. It isn't autogenerated I guess. The version from Keras 1 had static content for that particular doc.

**Comment by fchollet** [comment]
I just fixed it.
